apps/services/lib/service.ex

```diff
@@ -46,19 +50,24 @@ defmodule Services.Service do
           start_date: Date.t() | nil,
           type: type,
           typicality: typicality,
-          valid_days: [valid_day]
+          valid_days: [valid_day],
+          rating_start_date: Date.t() | nil,
+          rating_end_date: Date.t() | nil,
+          rating_description: String.t() | nil
```

Comment:
Given the default-to-empty-string logic below, it doesn't seem like this field _should_ ever be nil (and it looks like the type of the regular `description`, which has the same logic, is just listed as `String.t()`). Of course, we aren't using `@enforce_keys`, so  ¯\\\_(ツ)\_/¯ (it could be a larger tech debt project to make our enforced keys align with our typespecs)